tools/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
 add_subdirectory(p4mlir-translate)
 add_subdirectory(p4mlir-opt)
+add_subdirectory(p4mlir-export)
```

Comment:
Could we rename this to `p4mlir-export-bmv2` to be more specific?

Reply:
I feel it is a folder with options, inside this folder different emitters could emit protobuf, JSON, or even p4info, or bmv2 better?

Reply:
got you

Reply:
Done